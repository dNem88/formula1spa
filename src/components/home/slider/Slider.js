import React from 'react'

import styles from './slider.module.css'

function Slider(props) {

    return (
        <div className={styles.slider}>{props.children}</div>
    )
}

export default Slider