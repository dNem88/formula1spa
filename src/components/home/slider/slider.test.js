import React from 'react'
import renderer from 'react-test-renderer'
import Slider from './Slider'
import {MemoryRouter} from 'react-router-dom'


describe('Test Slider', () => {
    it('Should Render Properly Slider', () => {
        let slider = renderer.create(
            <MemoryRouter>
                <Slider />
            </MemoryRouter>
        ).toJSON()
        expect(slider).toMatchSnapshot()
        expect(slider.type).toBe('div')
        expect(slider.props.className).toBe('slider')
    })
})