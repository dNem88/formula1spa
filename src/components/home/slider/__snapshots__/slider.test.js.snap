// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Slider Should Render Properly Slider 1`] = `
<div
  className="slider"
/>
`;
